bestpractices/general_bestpractices.html.md.erb

```diff
@@ -9,6 +9,8 @@ When using HAWQ, adhere to the following guidelines for best results:
 -   **Use a consistent `hawq-site.xml` file to configure your entire cluster**:
 
     Configuration guc/parameters are located in `$GPHOME/etc/hawq-site.xml`. This configuration file resides on all HAWQ instances and can be modified by using the `hawq config` utility. You can use the same configuration file cluster-wide across both master and segments.
+    
+    If you install and manage HAWQ using Ambari, be aware that any property changes to `hawq-site.xml` made using the command line could be overwritten by Ambari. For Ambari-managed HAWQ clusters, always use the Ambari administration interface, not `hawq config`, to set or change HAWQ configuration properties. 
```

Comment:
Because we know that Ambari will overwrite local changes, I don't think we should say this "could" happen.  Maybe edit this note (globally) to be more succinct:

If you install and manage HAWQ using Ambari, do not use `hawq config` to set or change HAWQ configuration properties. Instead, use the Ambari interface for all configuration changes.  Any property changes that are made to `hawq-site.xml` outside of the Ambari interface will be overwritten when when you restart or reconfigure HAWQ with Ambari.